### PR TITLE
Modules directory in Windows + macOS replaced to Darwin

### PR DESCRIPTION
**Pull request by enty8080**
Support for Windows.
Defined OS Darwin, not macOS.

### CODE REVIEW

File: default/berry.c [hunk 1]
```diff
@@ -310,11 +310,17 @@ static int analysis_args(bvm *vm, int argc, char *argv[])
     return load_file(vm, argc, argv, args);
 }
 
-/* TODO: more paths & support more OS */
-#define BERRY_ROOT "/usr/local"
-static const char *module_paths[] = {
-    BERRY_ROOT "/lib/berry/packages",
-};
+#if defined(_WIN32)
+    #define BERRY_ROOT "/Windows/system32"
```
Comment (skiars) [inline default/berry.c]: Do you need a drive letter here, like `"C:/Windows/system32"`.

**Comment by enty8080** [comment]
@Skiars I fixed this, now it works. You do not need to define `C:\`, you can just use `\`

**Comment by skiars** [comment]
I got the error of escape character  (use GCC 8.1.0):

```
[Compile] default/berry.c
default/berry.c:316:16: error: unknown escape sequence: '\W'
     BERRY_ROOT "\berry\packages",
                ^~~~~~~~~~~~~~~~~
default/berry.c:316:16: error: unknown escape sequence: '\s'
default/berry.c:316:16: error: unknown escape sequence: '\p'
make: *** [Makefile:58: default/berry.o] Error 1
```

The backslash '\' should be escaped with "\\".

**Comment by enty8080** [comment]
@Skiars I do not know how to fix this :(

**Comment by enty8080** [comment]
@Skiars problem fixed.

**Pull request merged**